helper-cli/src/main/kotlin/commands/ListLicensesCommand.kt

```diff
@@ -181,11 +181,15 @@ internal class ListLicensesCommand : CliktCommand(
 private data class TextLocationGroup(
     val locations: Set<TextLocation>,
     val text: String? = null
-)
+) {
```

Comment:
Commit-message: What performance problem do we actually have and how much does it help?

Reply:
> What performance problem do we actually have and how much does it help?

I've improved the commit message in that regard. I did not benchmark the changes, though, as I believe they are beneficial even when leaving the performance aspect aside.

> _edit_: I'm asking because I find the code gets less readable in some places after this change which I know is subjective.

I only saw this edit after I've reworded the commit message. Looks like we again have different 
views, as I find the code to now be more readable throughout the changes.

Reply:
> I've improved the commit message in that regard. I did not benchmark the changes, though, as I believe they are beneficial even when leaving the performance aspect aside.

Thanks, this addressed actually my actually nit concerns. 

> I only saw this edit after I've reworded the commit message. Looks like we again have different views, as I find the code to now be more readable throughout the changes.

Sorry, I've mixed things up a bit with the other commit in this PR. I actually was referring to the readability change when using this type alias.

Reply:
> I actually was referring to the readability change when using this type alias.

Ah, ok. I agree introducing typealiases (only) for Path- and ScopeExcludes, and only for non-mutable but not for mutable lists, is a *bit* arbitrary. I initially made that change as some lines otherwise would have exceeded 120 chars, and they couldn't be wrapped nicely. But meanwhile I've changed the code anyway and I could drop the typealias change if you prefer.

Reply:
I would prefer to drop that commit, we use so many lists throughout the code that it's really inconsistent to add typealiases for two of the generic types, and I agree that it makes it harder to read the code, because you need to know the typealias.

Reply:
> I would prefer to drop that commit

Done, although ...

> because you need to know the typealias.

... this would mean we should *never* use typealiases, which I don't agree to.